chapter_deep-learning-basics/linear-regression-scratch.md

```diff
@@ -64,10 +64,12 @@ plt.scatter(features[:, 1].asnumpy(), labels.asnumpy(), 1);
 def data_iter(batch_size, features, labels):
```

Comment:
We use inline comments to keep each code block small: the book will be thinner :)
Can you revert changes in this md file?

Reply:
Oh... I see. I've restored the relevant changes.